datajoint/errors.py

```diff
@@ -81,6 +81,18 @@ class MissingExternalFile(DataJointError):
     """
 
 
+class DirectoryInaccessible(DataJointError):
```

Comment:
hm... this seems like a non-datajoint-specific error. Should we just raise `FileNotFoundError` instead of creating a datajoint-specific exception?